src/features/courses/course-controller.ts

```diff
@@ -1641,7 +1641,7 @@ class CourseController {
                             weight: parseInt(problem.value ?? '1'),
                             maxAttempts: parseInt(problem.max_attempts ?? '-1'),
                             hidden: false,
-                            optional: false,
+                            optional: problem.rederlyOptional === 'true'
```

Comment:
Personally, I like doing `true.toString()` to be consistent

Reply:
Coming from the guy that does `++i` for efficiency 🤣